oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java

```diff
@@ -57,11 +58,21 @@
     @Override public StreamObserver<ExportMetricsServiceRequest> export(
         StreamObserver<ExportMetricsServiceResponse> responseObserver) {
         return new StreamObserver<ExportMetricsServiceRequest>() {
-            @Override public void onNext(ExportMetricsServiceRequest request) {
+            private Node node;
+
+            @Override
+            public void onNext(ExportMetricsServiceRequest request) {
+                if (request.hasNode())
+                    node = request.getNode();
```

Comment:
I am not sure whether without `{}` is allowed. We will see.

Reply:
In case of NPE, I prefer to assign a default value to the `Node`, even node is sent in the first message.

Reply:
Do you mean build an node with mock values of fields we need?

Reply:
Exactly, `Optional.orElse()` is what I like. Some defensive programming style.